twoliter/embedded/build.Dockerfile

```
@@ -100,8 +100,11 @@ ARG ARCH
 ARG NOCACHE
 ARG VARIANT
 ARG SYSTEMD_NETWORKD
+ARG BUILD_ID
+ARG BUILD_ID_TIMESTAMP
 ENV SYSTEMD_NETWORKD=${SYSTEMD_NETWORKD}
 ENV VARIANT=${VARIANT}
+ENV BUILD_ID=${BUILD_ID} BUILD_ID_TIMESTAMP=${BUILD_ID_TIMESTAMP}
```

Did you mean for these to be on the same line? Does that have some meaning?

```suggestion
ENV BUILD_ID=${BUILD_ID}
ENV BUILD_ID_TIMESTAMP=${BUILD_ID_TIMESTAMP}
```

---

No reason in particular, just following some other conventions I'd seen in this Dockerfile. I can separate these onto two lines 👍